frontend/Views/NewDiscussion/actions.js

```diff
@@ -60,13 +60,13 @@ export const postDiscussion = (userId, forumId, currentForum) => {
       });
     }
 
-    if (tags === null || tags.length === 0) {
-      validated = false;
-      return dispatch({
-        type: POSTING_DISCUSSION_FAILURE,
-        payload: 'Please provide some tags.',
-      });
-    }
+    // if (tags === null || tags.length === 0) {
+    //   validated = false;
+    //   return dispatch({
+    //     type: POSTING_DISCUSSION_FAILURE,
+    //     payload: 'Please provide some tags.',
+    //   });
+    // }
```

Comment:
any reason we want to comment this instead of deleting it? If we ever need it later it's still in git.

Reply:
This is addressed in another PR: https://github.com/yubodiwu/ReForum/pull/24